pkg/controller/directvolumemigration/rsync.go

```diff
@@ -565,6 +576,65 @@ func (t *Task) getRsyncRoute(namespace string) (string, error) {
 	return route.Spec.Host, nil
 }
 
+func (t *Task) createRsyncPassword(destClient compat.Client) error {
+	var letters = []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	random.Seed(time.Now().UnixNano())
+	password := make([]byte, 6)
+	for i := range password {
+		password[i] = letters[random.Intn(len(letters))]
+	}
+
+	secret := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: migapi.OpenshiftMigrationNamespace,
+			Name:      "rsync-pass",
+			Labels: map[string]string{
+				"app": "directvolumemigration-rsync-transfer",
+			},
+		},
+		Data: map[string][]byte{
+			"password": password,
+		},
+	}
+	err := destClient.Create(context.TODO(), &secret)
+	if k8serror.IsAlreadyExists(err) {
+		t.Log.Info("Secret already exists on destination", "namespace", migapi.OpenshiftMigrationNamespace)
+	} else if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (t *Task) getRsyncPassword(destClient compat.Client) ([]byte, error){
+	rsyncSecret := corev1.Secret{}
+	key := types.NamespacedName{Name: "rsync-pass", Namespace: migapi.OpenshiftMigrationNamespace}
+	err := destClient.Get(context.TODO(), key, &rsyncSecret)
+	if err != nil {
+		return nil, err
+	}
+	return rsyncSecret.Data["password"], nil
```

Comment:
if the password does not exist, this could panic?  probably worth hiding this behind a check

```
    if pass, ok := rsyncSecret.Data["password"]; ok {
        return pass, nill
    }
    return "", nil
```

Reply:
+1